server/paymentProviders/opencollective/prepaid.js

```diff
@@ -80,9 +80,17 @@ async function processOrder(order) {
     throw new Error("This payment method doesn't have enough funds to complete this order");
   }
 
+  let hostFeeInHostCurrency, platformFeeInHostCurrency;
+
+  if (options.skipPlatformFee && options.skipHostFee) {
```

Comment:
That doesn't make sense. You should handle these properties individually.

Reply:
Oh thought they were together. I'll change it